### ISSUE TITLE
When I trained, I encountered cuDNN error. Is it a problem with tensorflow?

### ISSUE THREAD

**Issue by Jai-wei** [qqwweee/keras-yolo3#630]
UnknownError (see above for traceback): Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[node conv2d_1/convolution (defined at /home/magic/anaconda3/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3335) ]]
	 [[node loss/add_74 (defined at /home/magic/anaconda3/lib/python3.6/site-packages/keras/engine/training.py:848) ]]

**Comment by Jai-wei** [comment]
cuDNN：

magic@magic:~$ cat /usr/local/cuda/include/cudnn.h | grep CUDNN_MAJOR -A 2
#define CUDNN_MAJOR 7
#define CUDNN_MINOR 5
#define CUDNN_PATCHLEVEL 0
--
#define CUDNN_VERSION (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 + CUDNN_PATCHLEVEL)
#include "driver_types.h"

CUDA:  10.0
tensorflow:   1.13.1

**Comment by dzb1992** [comment]
You can change the dynamically allocated memory and add the following code at the beginning of the code:
`import tensorflow as tf`
`import keras`
`config = tf.ConfigProto()`
`config.gpu_options.allow_growth = True`
`keras.backend.tensorflow_backend.set_session(tf.Session(config=config))`

**Comment by MykolaRobot** [comment]
I had similar issue and I fixed it by installing tensorflow-gpu==1.14.0
